_data/toc.yaml

```diff
@@ -329,50 +355,22 @@ guides:
         title: Overlay networking tutorial
       - path: /network/network-tutorial-macvlan/
         title: Macvlan network tutorial
-    - sectiontitle: Configure the daemon and containers
-      section:
-      - path: /config/daemon/ipv6/
-        title: Configure the daemon for IPv6
-      - path: /network/iptables/
-        title: Docker and iptables
-      - path: /config/containers/container-networking/
-        title: Container networking
-      - path: /network/proxy/
-        title: Configure Docker to use a proxy server
     - sectiontitle: Legacy networking content
       section:
       - path: /network/links/
         title: (Legacy) Container links
-  - sectiontitle: Manage application data
+  - sectiontitle: Configure storage
     section:
     - path: /storage/
-      title: Storage overview
+      title: Overview
     - path: /storage/volumes/
       title: Volumes
     - path: /storage/bind-mounts/
       title: Bind mounts
     - path: /storage/tmpfs/
       title: tmpfs mounts
     - path: /storage/troubleshooting_volume_errors/
-      title: Troubleshoot volume problems
-    - sectiontitle: Store data within containers
-      section:
-      - path: /storage/storagedriver/
-        title: About storage drivers
-      - path: /storage/storagedriver/select-storage-driver/
-        title: Select a storage driver
-      - path: /storage/storagedriver/aufs-driver/
-        title: Use the AUFS storage driver
-      - path: /storage/storagedriver/btrfs-driver/
-        title: Use the Btrfs storage driver
-      - path: /storage/storagedriver/device-mapper-driver/
-        title: Use the Device mapper storage driver
-      - path: /storage/storagedriver/overlayfs-driver/
-        title: Use the OverlayFS storage driver
-      - path: /storage/storagedriver/zfs-driver/
-        title: Use the ZFS storage driver
-      - path: /storage/storagedriver/vfs-driver/
-        title: Use the VFS storage driver
+      title: Troubleshoot
```

Comment:
@thaJeztah Notes from my meeting with you suggests that discussed to move the Troubleshoot section to Bind mounts. Looks like the troubleshoot topic contains info on both bind mounts and volumes. Wondering if the move still makes sense. Could you confirm? Thanks.